catalog/ui/catalog-ui-search/pom.xml

```diff
@@ -73,6 +73,10 @@
             <version>${camel.version}</version>
         </dependency>
         <dependency>
+            <groupId>org.json</groupId>
```

Comment:
Also get rid of this dependency and on line 322 if you replaced the usages with boon

Reply:
removed from pom